listenbrainz/webserver/errors.py

```diff
@@ -57,41 +57,61 @@ def error_wrapper(template, error, code):
         resp.headers['Access-Control-Allow-Origin'] = '*'
         return resp, code
 
+    def json_error_wrapper(error, code):
+        return jsonify({
+            'code': code,
+            'error': error.description,
+            }), code
+
+    def handle_error(error, code):
+        """ Returns appropriate error message on HTTP exceptions
+
+            error (werkzeug.exceptions.HTTPException): The exception that needs to be handled
+            code (int): the HTTP error code that should be returned
+
+            Returns:
+                A Response which will be a json error if request was made to the LB api and an html page
+                otherwise
+        """
+        if request.path.startswith('/1/'):
+            return json_error_wrapper(error, code)
+        else:
+            return error_wrapper(f'errors/{code}.html', error, code)
```

Comment:
It looks a bit better `%s` and `.format` to me, and since CB has decided to move to using f-strings, I thought slowly moving to them here would be good.